src/common/drain/drain.js

```diff
@@ -55,7 +55,7 @@ export function drain (agentIdentifier = '', featureName = 'feature') {
 
   // Only when the event-groups for all features are ready to drain (staged) do we execute the drain. This has the effect
   // that the last feature to call drain triggers drain for all features.
-  const items = Array.from(registry[agentIdentifier])
+  const items = Array.prototype.slice.call(registry[agentIdentifier])
```

Comment:
This is not going to work to convert a Set to an Array

Reply:
Ah-ha it is a set. I was wondering about that. Yea this will break stuffs.